### ISSUE TITLE
Usage remote images

### ISSUE THREAD

**Issue by luforestal** [flowerchecker/kindwise-api-client#1]
Hello, 

I'm trying to use with images located in remote (access with a URL) in python 3.11. I'm using POST method, but I can't access. Have you an example with a consultation with URL?

**Comment by simonplhak** [comment]
Hello,

SDK currently does not support providing image as an URL, because strings are considered by SDK as a path to an image or directly encoded image in base64. You can use the following snippet as a roundabout for now; we'll try to add this behavior to our SDK in future releases.
```
import base64

import requests

from kindwise import PlantApi, PlantIdentification

image_url = 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQ8oP-mTzEfB3SAw9lkKKSZJhfD123tEvcbjmi-H5ftfzMBBGhd'
response = requests.get(image_url)
# Encode the image in base64
img = base64.b64encode(response.content)
api = PlantApi()
identification: PlantIdentification = api.identify(img, details='common_names')
print(identification)
```

**Comment by simonplhak** [comment]
Solved in version 0.4.1
